.github/workflows/build-linux.yml

```diff
@@ -28,3 +28,8 @@ jobs:
         name: server-logs
         path: '**/server.log*'
         retention-days: 3
+    - name: Publish Test Report
+      uses: mikepenz/action-junit-report@v5
+      if: success() || failure() # always run even if the previous step fails
```

Comment:
Why not `always()` and no comment? (I find the current comment as misleading a bit as it says _always_, but `cancelled` is not covered...)